### ISSUE TITLE
AutoComplete does not show up when opening a dialog; only after saving a note

### ISSUE THREAD

**Issue by MichaelKim0407** [AnKing-VIP/anki-field-autocomplete#17]
#### Problem description

Not sure if this is the same as #14 or #16, but I have reliably reproduced this behavior that I'm reporting. Sorry for a potential duplicate.

When I first open the `Add` dialog, the `A` button as well as autocomplete dropdown do not appear. They appear and start working after saving a note. This happens every time I close and open the `Add` dialog, regardless of how long Anki has been open.

The same happens for the editing interface in the `Browse` dialog. The first time it appears, there is no `A` or autocomplete dropdown, however it appears as soon as I click on another card or note.

#### Checklist

*Please replace the space inside the brackets with an **x** if the following items apply:*

 - [x] I've restarted Anki to see if it helps
 - [x] I've verified that I use the latest version of the add-on by redownloading it from AnkiWeb
 - [x] I've tried to disable other add-ons to see if there are any interactions present
 - [ ] My issue disappears when I hold shift while starting Anki.
 - [x] I've checked if anyone else reported this problem before by looking through the issue reports. I also checked to see if there is a section about known issues in the add-on description, documentation, or README.
 

#### What Anki version are you using?

*Please open Anki, go to Help → About (windows) or Anki → About Anki (mac) to find the version. Edit the value between the backticks below

```
Version ⁨24.04.1 (ccd9ca1a)⁩
Python 3.9.18 Qt 6.6.2 PyQt 6.6.1
```

#### Error message (if any)

*If you've received an error message, please copy and paste it between the backticks below:*

```

```

**Comment by MichaelKim0407** [comment]
Seems to have been fixed by the new update, thanks!
